cpp/src/arrow/symbols.map

```diff
@@ -55,6 +55,8 @@
     ERR_getErrorString;
     # jemalloc
     je_arrow_*;
+    # ORC destructors
+    _ZThn8_N3orc*;
```

Comment:
This is very odd! These aren't captured by normal C++ mangling?

Reply:
No, they (non-virtual destructors) get ignored by the C++ mangling below.